src/olympia/addons/tests/test_tasks.py

```diff
@@ -92,13 +92,13 @@ def test_create_missing_theme_previews(parse_addon_mock):
     assert VersionPreview.objects.count() == 3
     with mock.patch(f'{PATCH_PATH}.generate_static_theme_preview') as gstp, mock.patch(
         f'{PATCH_PATH}.resize_image'
-    ) as rs:
+    ) as resize:
         recreate_theme_previews([theme.id], only_missing=True)
         assert gstp.call_count == 0
```

Comment:
Renaming `gstp` would be nice, too

Reply:
alternate name that doesn't overflow the line length and result in even more confusing over-formatting welcome 😉

Reply:
`gen_preview`?